src/server/index.js

```diff
@@ -175,6 +177,40 @@ app.post('/api/businesses', function(req, res) {
   }
 });
 
+app.post('/api/business/:id/comments', jwtMiddleware,
```

Comment:
modify this to `/api/businesses/:id/comments` or discuss with the team that we use singular nouns for the resources